### DISCUSSION TITLE
Only the Database is Being Backed Up, Not the Files

### DISCUSSION THREAD

**Discussion by HartFinne** [spatie/laravel-backup#1855] [Q&A]
I have followed the installation and backup instructions provided by Spatie, but when attempting to run the backup command, the resulting folder is empty. I even tried using the php artisan backup:run --only-files command, but it still returned a folder with nothing inside. Could someone kindly assist me in resolving this issue?

Thank you in advance!

![image](https://github.com/user-attachments/assets/c0a8157a-b529-4031-b246-c08c82713782)
![image](https://github.com/user-attachments/assets/c9a46b11-1f11-472e-aa49-4e740c11b8b1)

**Comment by 4unkur** [comment]
@HartFinne please share your config/backup.php file
